recipes/libwebsockets/all/conanfile.py

```diff
@@ -424,7 +421,7 @@ def _configure_cmake(self):
 
 
 
-        self._cmake.configure(build_folder=self._build_subfolder)
+        self._cmake.configure()
```

Comment:
Why removing build_folder? Any forced cmake path in cmakelists?

Reply:
to decrease build path length, otherwise CCI fails:
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppBuild.targets(321,5): error MSB3491: Could not write lines to file "websockets_shared.dir\Release\websocke.43D53957.tlog\websockets_shared.lastbuildstate". The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters. [C:\J\w\cci_PR-2707\1\5ed19867-2b6b-4545-af54-d490092d6233\.conan\data\libwebsockets\4.0.15\_\_\build\a9ab68c1931cd1e853c1c8797538398b92266fcb\build_subfolder\source_subfolder\websockets_shared.vcxproj]
```

Reply:
probably it's a limitation using Jenkins (I guess). Anyway, that's a good workaround.